### DISCUSSION TITLE
✅ Ability to enable/disable widgets and features in Elementor - [ED-12546]

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#21717] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi elementor team.

I believe Elementor should add two new sections called **widgets** and **features** in its settings. 

![new](https://user-images.githubusercontent.com/88034902/226561111-100c7dee-a0dc-445d-92ff-3c48c16ff88e.png)

widgets : image - menu - loop & all widgets.

features : Form Submissions - Save as Default - Additional Custom Breakpoints & all features. This is something that will be a game changer. Because we can enable/disable all elementor widgets and features. If you want a full sample of this offer, check out the Powerpack plugin.

### Describe the solution you'd like

Ability to enable/disable widgets and features in Elementor

**Comment by m1ga** [comment]
+1 for the widget part! Would clean up the menu a you could remove widgets that a user shouldn't be able to use 👍

**Comment by acornavi** [comment]
Automatic removal of the unnecessary code from the page of the unused widgets would be very helpful as well :)

**Reply by ChasHenrySEO** [reply]
100%. Right now we are disabling with third party tools and it can be time consuming on huge sites depending on what's being used on each page.

**Comment by amirhoseinsadeghi** [comment]
+1

**Comment by ayoubkhan558** [comment]
+1

**Comment by ayoubkhan558** [comment]
### Also, add the ability to scan unused widgets :) +111

**Comment by ghost** [comment]
### Please change the title to " Ability to scan enable/disable widgets and features in Elementor"

**Comment by arielk** [comment]
Ref: https://github.com/elementor/elementor/issues/9647

**Comment by nicholaszein** [comment]
Hi! 👋 

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor v3.18.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
